dogs/src/app/home-page/home-page.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from "@angular/core";
+
+import { DogService } from "../shared/api-dog.service";
+
+@Component({
+	selector: "app-home-page",
+	templateUrl: "./home-page.component.html",
+	styleUrls: ["./home-page.component.css"]
+})
+export class HomePageComponent implements OnInit {
+
+	public dogBreeds: string[] = [];
```

Comment:
```suggestion
	public breedNames: string[] = [];
```
stick to the convention